### PR TITLE
dockerfile, build system, failing acceptance test, passing dummy unit…

### PR DESCRIPTION
**Pull request by ztbrown**
… test

### CODE REVIEW

**Comment by ztbrown** [comment]
@weirichd: check it

File: tests/Makefile [hunk 1]
```diff
@@ -0,0 +1,22 @@
+TARGET = test_suite
+SRCS = test_suite.c acceptance_test_library.c 
+OBJS = $(SRCS:.c=.o)
+LIB=../lib/libmock.a
+
+CFLAGS=-std=c99 -Wall
+
+$(info,$(SRCS) $(OBJS))
```
Comment (weirichd) [inline tests/Makefile]: Do we need this?

File: tests/Makefile [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+all: $(TARGET)
+
+%.o: %.c
+	gcc $(CFLAGS) -c -o $@ $^
+
+$(TARGET): $(OBJS) $(LIB)
+	gcc $(CFLAGS) -g -o $@ $^ `pkg-config --cflags --libs check`
```
Comment (weirichd) [inline tests/Makefile]: Maybe include the `-g` flag in with `CFLAGS`, or just omit it until we need it.

**Review by weirichd — COMMENTED** [review]
Don't commit the binary file.

**Comment by ztbrown** [comment]
TFW you've checked in the binary:
gaucxpf

File: tests/Makefile (hunk 1 above)
Comment (ztbrown) [reply tests/Makefile]: No. We can always add it back if we want the debug information.

**Pull request merged**